tox.ini

```diff
@@ -19,55 +19,53 @@ norecursedirs = .* docs requirements venvs
 
 [testenv]
 deps = 
-	django22: Django>=2.2,<2.3
-	django30: Django>=3.0,<3.1
-	django31: Django>=3.1,<3.2
-	django32: Django>=3.2,<3.3
-	-r{toxinidir}/requirements/test.txt
+    django32: Django>=3.2,<4.0
+    django40: Django>=3.3,<4.1
+    -r{toxinidir}/requirements/test.txt
 whitelist_externals = 
-	touch
-	rm
```

Comment:
you should undo these indentation changes

Reply:
As discussed in this [conversation](https://github.com/openedx/edx-rest-api-client/pull/192#discussion_r786451260), these changes are one time and wouldn't happen again on next run